lib/builder.js

```diff
@@ -143,8 +143,11 @@ module.exports = class Builder extends EventEmitter {
           this.buildHeimdallTree(outputNodeWrapper);
         }),
       () => {
-        let buildsSkipped = this.nodeWrappers.filter(nw => nw.buildState.built === false).length;
-        logger.debug(`Total nodes skipped: ${buildsSkipped} out of ${this.nodeWrappers.length}`);
+        let buildsSkipped = filterMap(
```

Comment:
Note for a future iteration: it would be really nice if heimdall-logger provided a way to _conditionally_ emit logging info. Right now we are forced to eagerly calculate the debugging output when in 99% of the time its never consumed / reviewed.